From 832fad953dc7779d4e7631f35ba2fed7dbbe49a3 Mon Sep 17 00:00:00 2001
From: Simon Xue <xxm@rock-chips.com>
Date: Mon, 24 Dec 2018 14:16:34 +0800
Subject: [PATCH] arm64: dts: rockchip: rk1808: add power-domains reference for
 PCIe

RK1808 PCIe share pd with others, add pd reference in case being
turned off by others

Change-Id: I899eb7524ae9aaada3cc161a63434113a3a1768e
Signed-off-by: Simon Xue <xxm@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index c4264cafa889..c1e7ddaf67dc 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -186,6 +186,7 @@
 		phy-names = "pcie-phy";
 		pinctrl-names = "default";
 		pinctrl-0 = <&pcie_clkreq>;
+		power-domains = <&power RK1808_PD_PCIE>;
 		ranges = <0x00000800 0x0 0xf8000000 0x0 0xf8000000 0x0 0x800000
 			  0x83000000 0x0 0xf8800000 0x0 0xf8800000 0x0 0x3700000
 			  0x81000000 0x0 0xfbf00000 0x0 0xfbf00000 0x0 0x100000>;
-- 
2.35.3

